packages/site/cms-site/src/sitePreview/SitePreviewUtils.ts

```diff
@@ -0,0 +1,63 @@
+import jsonwebtoken from "jsonwebtoken";
```

Comment:
Will any of this be imported in client code? This would add jsonwebtoken to the client bundle, causing around 100kb.

Reply:
yes, it is in client code :( (not 100kb though)

that is difficult to solve, I'd defer that for later (let's discuss the issue tomorrow)

Reply:
Additional Info why I want to ignore this for now:

- the whole file SitePreviewUtils must not run on the client (as it doesn't have access to the secret)
- we should add `import "server-only";`
- currently this doesn't work, tree shaking our `@comet/cms-site` barrel index file currently doesn't work :/
- fixing that is out of scope for this PR

Reply:
```suggestion
//TODO add import "server-only"; once this file gets correctly tree-shaked out of the client bundle

import jsonwebtoken from "jsonwebtoken";
```